### ISSUE TITLE
Issue with initial setup

### ISSUE THREAD

**Issue by SyedShaQutub** [ivankreso/ladder-densenet#4]
Hello everyone,

I have downloaded the cityscapes gtFine data set. 

1. Is it the right dataset because I don't see the structure similar to as it is defined in the code with subfolders as "datasets/Cityscapes/2048x1024/"

While preparing the dataset by running the script python "prepare_dataset.py", I encounter several issues. I believe people posted issues here, have already surpassed this stage. Hope you could help me with this.

2. Many functions are not called from the right directory - did anyone face this issue?
3. Issue in file prepare_dataset.py (cityscapes folder line 91) -  
error: cities = next(os.walk(root_dir))[1]
StopIteration
4. I'm not sure about the optional command --data_dir=/path/to/data --gt_dir=/path/to/labels --save_dir=/path/to/save

Thank you.

**Comment by joonjeon** [comment]
Same issue here...I am trying to verify the accuracy and performace of this model and have encountered multiple issues similar to those that @SyedShaQutub encountered.

1. It seems like the original developers simply set up their own custom directory structure for preparing their dataset and training. So, I modified the training dataset and ground truth directories to my own ones.

2. In line 2 of `cityscapes.py`, it is mentioned as `from datasets.dataset import Dataset`, but I think this should be changed to `from dataset import Dataset`. Otherwise, the ImportError error occurs and the code will not run at all.

3. **The StopIteration error is the one that I am still  having trouble with.**  I have no idea why this is occurring, and it seems the original authors of the model must reply to this issue. Otherwise I cannot execute the code at all.

4. It looks like the data_dir refers to the directory in which the original image data is included, and gt_dir refers to the directory with ground truth labels. The save_dir seems to indicate the directory in which to save the output training results.
